src/content/en/2024/third-parties.md

```diff
@@ -3,17 +3,205 @@
 title: Third Parties
 description: Third Parties chapter of the 2024 Web Almanac covering data of what third parties are used on the web and a deep dive into preventing them causing negative performance impacts.
 hero_alt: Hero image of Web Almanac characters plugging various things into a web page.
-authors: []
-reviewers: []
-editors: []
-analysts: []
+authors: [turban1988, Yash-Vekaria, zubairshafiq, ChrisBeeti]
+reviewers: [tunetheweb]
+editors: [neriiavr]
+analysts: [Yash-Vekaria, ChrisBeeti]
 translators: []
+turban1988_bio: TODO
+Yash-Vekaria_bio: TODO
```

Comment:
@Yash-Vekaria can you please provide a short bio about yourself?

Reply:
Yash Vekaria is a PhD candidate in Computer Science at [University of California, Davis](https://www.ucdavis.edu). He carries out web-based large-scale Internet measurements to study and improve the dynamics of web. Specifically, his research is focused at studying and bringing transparency to online tracking practices and user privacy issues.

Reply:
```suggestion
Yash-Vekaria_bio: Yash Vekaria is a PhD candidate in Computer Science at <a hreflang="en" href="https://www.ucdavis.edu/">University of California, Davis</a>. He carries out web-based large-scale Internet measurements to study and improve the dynamics of web. Specifically, his research is focused at studying and bringing transparency to online tracking practices and user privacy issues.
```